src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -171,22 +187,16 @@ def _failed_handler(self, events: List[CloudEvent]) -> None:
         # api for setting state in the ensemble
         if len(events) == 0:
             events = [self._create_cloud_event(EVTYPE_ENSEMBLE_FAILED)]
-        with self._snapshot_mutex:
-            snapshot_update_event = self.ensemble.update_snapshot(events)
-        send_future = asyncio.run_coroutine_threadsafe(
-            self._send_snapshot_update(snapshot_update_event), self._loop
-        )
-        send_future.result()
-        self._signal_cancel()  # let ensemble know it should stop
+        await self._append_message(self.ensemble.update_snapshot(events))
+        await self._signal_cancel()  # let ensemble know it should stop
 
-    async def _send_snapshot_update(
-        self, snapshot_update_event: PartialSnapshot
-    ) -> None:
-        message = self._create_cloud_message(
-            EVTYPE_EE_SNAPSHOT_UPDATE,
-            snapshot_update_event.to_dict(),
-        )
-        await self._send_message(message)
+    @property
+    def config(self) -> EvaluatorServerConfig:
```

Comment:
This property is unused in the tests.

Reply:
checking if it is alright to remove this too.